File: KnowledgeSpaceApk/adminapp/src/main/java/com/example/adminapp/AlumniActAdm/AlumniActMainScAdm.java
package com.example.adminapp.AlumniActAdm;

import androidx.appcompat.app.AppCompatActivity;
import androidx.fragment.app.FragmentManager;
import androidx.fragment.app.FragmentTransaction;

import android.os.Bundle;

import com.example.adminapp.R;

public class AlumniActMainScAdm extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_alumni_act_main_sc_adm);
        FragmentManager fragmentManager = getSupportFragmentManager();
        FragmentTransaction fragmentTransaction = fragmentManager.beginTransaction();
        fragmentTransaction.replace(R.id.frameLayAlumniActAdm,new AlumniFragHomeSc());
        fragmentTransaction.commit();
    }
}

File: KnowledgeSpaceApk/adminapp/src/main/java/com/example/adminapp/AlumniActAdm/AlumniActMainScAdm.kt
package com.example.adminapp.AlumniActAdm

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import com.example.adminapp.R

class AlumniActMainScAdm : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_alumni_act_main_sc_adm)
        val fragmentManager = supportFragmentManager
        val fragmentTransaction = fragmentManager.beginTransaction()
        fragmentTransaction.replace(R.id.frameLayAlumniActAdm, AlumniFragHomeSc())
        fragmentTransaction.commit()
    }
}